capstone/src/main/webapp/js/calendar.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Callback function to generate charts.
+ */
+function drawCharts() {
+  const freqData = new google.visualization.DataTable();
```

Comment:
What about frequencyData instead of freqData?
Google doesn't favor abbreviations in names.

Reply:
Ah yea that works, will switch to `frequencyData` from `freqData`.